Makefile

```diff
@@ -18,54 +18,75 @@
-
-#########
-#  App  #
-#########
 
-APP_LOAD_PARAMS = --curve ed25519
-APP_LOAD_PARAMS += --appFlags 0x200  # APPLICATION_FLAG_BOLOS_SETTINGS
-APP_LOAD_PARAMS += --path "44'/3030'"
-APP_LOAD_PARAMS += $(COMMON_LOAD_PARAMS)
+include $(BOLOS_SDK)/Makefile.defines
 
-APPVERSION_M = 1
-APPVERSION_N = 4
-APPVERSION_P = 2
-APPVERSION = $(APPVERSION_M).$(APPVERSION_N).$(APPVERSION_P)
+########################################
+#        Mandatory configuration       #
+########################################
+# Application name
 APPNAME = Hedera
 
-COIN = HBAR
-
-DEFINES += $(DEFINES_LIB)
+# Application version
+APPVERSION_M = 1
+APPVERSION_N = 5
+APPVERSION_P = 0
+APPVERSION = "$(APPVERSION_M).$(APPVERSION_N).$(APPVERSION_P)"
 
+# Application source files
+APP_SOURCE_PATH  += src proto
+SDK_SOURCE_PATH  += lib_u2f
+
+# Application icons
+ICON_NANOS = icons/nanos_app_hedera.gif
+ICON_NANOX = icons/nanox_app_hedera.gif
+ICON_NANOSP = icons/nanox_app_hedera.gif
+ICON_STAX = icons/stax_app_hedera.gif
+ICON_FLEX = icons/flex_app_hedera.gif
+
+# Application allowed derivation curves.
+CURVE_APP_LOAD_PARAMS = ed25519
+
+# Application allowed derivation paths.
+PATH_APP_LOAD_PARAMS = "44'/3030'"   # purpose=coin(44) / coin_type=Hedera HBAR(3030)
+
+VARIANT_PARAM = COIN
+VARIANT_VALUES = hedera
+
+# Enabling DEBUG flag will enable PRINTF and disable optimizations
+#DEBUG = 1
+
+########################################
+#     Application custom permissions   #
+########################################
+HAVE_APPLICATION_FLAG_BOLOS_SETTINGS = 1
+
+########################################
+# Application communication interfaces #
+########################################
+ENABLE_BLUETOOTH = 1
+
+########################################
+#         NBGL custom features         #
+########################################
+ENABLE_NBGL_QRCODE = 1
+
+########################################
+#          Features disablers          #
+########################################
+# These advanced settings allow to disable some feature that are by
+# default enabled in the SDK `Makefile.standard_app`.
+DISABLE_STANDARD_APP_FILES = 1
+
+########################################
+#          App specific configuration  #
+########################################
 ifeq ($(TARGET_NAME),TARGET_NANOS)
-ICONNAME=icons/nanos_app_hedera.gif
-else ifeq ($(TARGET_NAME),TARGET_STAX)
-ICONNAME=icons/stax_app_hedera.gif
-else
-ICONNAME=icons/nanox_app_hedera.gif
+DISABLE_STANDARD_BAGL_UX_FLOW = 1
 endif
 
-
-################
-# Default rule #
-################
-
-all: default
-
-############
-# Platform #
-############
-
-DEFINES   += OS_IO_SEPROXYHAL
-DEFINES   += HAVE_SPRINTF
-
-DEFINES   += HAVE_IO_USB HAVE_L4_USBLIB IO_USB_MAX_ENDPOINTS=6 IO_HID_EP_LENGTH=64 HAVE_USB_APDU
-DEFINES   += APPVERSION_M=$(APPVERSION_M) APPVERSION_N=$(APPVERSION_N) APPVERSION_P=$(APPVERSION_P)
-
 # vendor/ledger-nanopb
-DFEFINES  += PB_FIELD_32BIT=1
+DEFINES  += PB_FIELD_32BIT=1
```

Comment:
If it was ok with previous wrongly written `DFEFINES`, could it be removed?

Reply:
Good question. I can take a look at the proto configuration stuff, although it is not the purpose of this PR to update these kinds of things.

Reply:
No, effectuively, but doing some cleanup, it is not necessary to keep a flag if it is useless... :thinking:

Reply:
I'm removing it. It compiles without it and the documentation says that the proto compiler should warn if the flag is needed (it doesn't in our case) : 

`PB_FIELD_32BIT: Add support for field tag numbers over 65535, fields larger than 64 kiB and arrays larger than 65535 entries. Compiler warning will tell if you need this.`